validators/signup/update.js

```diff
@@ -0,0 +1,20 @@
+const Joi = require('joi');
+
+const updateValidator = Joi.object({
+    firstname: Joi.string()
+        .min(3)
+        .max(30)
+        .required(),
+    lastname: Joi.number().required(),
+    age: Joi.number().required(),
+    email: Joi.string().required(),
+    date_of_birth: Joi.string().required(),
+    password: Joi.string().required(),
```

Comment:
add regex validation to make it more difficult to put simple password.
the min password should be 6 and max should be 10